docs/redoc/master/openapi.json

```diff
@@ -5204,18 +5204,29 @@
         }
       },
       "GeoPolygon": {
-        "description": "Geo filter request\n\nMatches coordinates inside the polygon, defined by the given coordinates in order.",
+        "description": "Geo filter request\n\nMatches coordinates inside the polygon, defined by exterior and interiors.",
         "type": "object",
         "required": [
-          "points"
+          "exterior",
+          "interiors"
         ],
         "properties": {
-          "points": {
-            "description": "Ordered list of coordinates representing the vertices of a polygon.",
+          "exterior": {
```

Comment:
With this design we are not respecting strictly the [GeoJSON API](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.6).

```
o  For type "Polygon", the "coordinates" member MUST be an array of
      linear ring coordinate arrays.

   o  For Polygons with more than one of these rings, the first MUST be
      the exterior ring, and any others MUST be interior rings.  The
      exterior ring bounds the surface, and the interior rings (if
      present) bound holes within the surface.
````

I like your proposal better as it is more explicit.

Reply:
Got you! Will change to rings and have it also align with gRPC.

Reply:
I have aligned both gRPC and REST to the GeoJSON API. Currently, I've named the field in the Polygon as rings, which is directly derived from the GeoJSON specification.